### DISCUSSION TITLE
import pytorch_lightning fails with ModuleNotFoundError: No module named 'tensorboard'

### DISCUSSION THREAD

**Discussion by Rafael8830** [Lightning-AI/pytorch-lightning#12002] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
### BUG
I finished installing pytorch_lightnin by pip but can not import pytorch_lightning.

### Environment

- CUDA: 11.2
- GPU: v100 * 8
- Packges:
-OpenCC==1.1.0
-pytorch-lightning==1.1.2
-six==1.14.0
-tensorboard==2.4.0
-tensorboard-plugin-wit==1.7.0
-threadpoolctl==2.1.0
-tokenizers==0.9.4
-torch==1.10.2+cu113
-transformers==4.1.1
-yacs
-lxml

- Error message
![image](https://user-images.githubusercontent.com/47704881/154795220-cab9b91e-73df-4c4e-b446-53e17125568a.png)

**Comment by akihironitta** [comment, marked as answer]
@Rafael8830 Could you try reinstalling it? If the issue still persists, could you provide more info about your environment?

**Reply by Rafael8830** [reply]
I reinstalled and the problem was solved. Thanks.
